### PR TITLE
sync website change

### PR DESCRIPTION
**Pull request by helen-frank**
…nsufficient.

#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
sync website change
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
According to the context. The word should be "underspecified" meaning "insufficient"
But, According https://github.com/kubernetes/website/pull/2991/files，I think a change like that is more reasonable.
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -154,8 +154,7 @@ via a flag.
 HTTP endpoint: HTTP endpoint passed as a parameter on the command line. This endpoint
 is checked every 20 seconds (also configurable with a flag).
 
-HTTP server: The kubelet can also listen for HTTP and respond to a simple API
-(underspec'd currently) to submit a new manifest.`,
+HTTP server: The kubelet can also listen for HTTP requests and respond to a simple API call to submit a new manifest.`,
```
Comment (SergeyKanzhelev) [inline cmd/kubelet/app/server.go:157]: is it even true? I cannot find pointers. I see `NewSourceFile`, `NewSourceURL`, `NewSourceApiserver`, but not the server. I don't know the history here.

Can you please check if this is even true. If it is not, let's remove this statement everywhere

Comment (helen-frank) [reply cmd/kubelet/app/server.go:157]: Looks like `kubelet.md` was moved [content/en/docs/reference/command-line-tools-reference/kubelet.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/kubelet.md)
but, the description here does not seem to be consistent with `https://github.com/kubernetes/website/pull/299`

Comment (helen-frank) [reply cmd/kubelet/app/server.go:157]: It looks like this document was lost during the document migration.

**Comment by SergeyKanzhelev** [comment]
See https://github.com/kubernetes/kubernetes/pull/3015

This whole functionality got removed from kubelet. So please remove the entire statement

File: cmd/kubelet/app/server.go [hunk 2]
```diff
@@ -152,10 +152,7 @@ periodically for updates. The monitoring period is 20s by default and is configu
 via a flag.
 
 HTTP endpoint: HTTP endpoint passed as a parameter on the command line. This endpoint
-is checked every 20 seconds (also configurable with a flag).
-
-HTTP server: The kubelet can also listen for HTTP and respond to a simple API
-(underspec'd currently) to submit a new manifest.`,
+is checked every 20 seconds (also configurable with a flag).`,
```
Comment (SergeyKanzhelev) [inline cmd/kubelet/app/server.go:155]:
```suggestion
is checked every 20 seconds (also configurable with a flag).`.
```

Comment (SergeyKanzhelev) [inline cmd/kubelet/app/server.go:155]: Also change the "there are three ways that a container" to "there are two ways that a container" on line 147

**Comment by SergeyKanzhelev** [comment]
/triage accepted
/priority backlog

I will ok-to-test once comments are addressed to not run tests unnecessary

**Comment by SergeyKanzhelev** [comment]
Also can you please send similar PRs to website repository so I can comment on it too. Thank you!

File: cmd/kubelet/app/server.go (hunk 2 above)
Comment (helen-frank) [reply cmd/kubelet/app/server.go:155]: I'm a little confused, this location `,` should exist as a separator in golang syntax

**Comment by helen-frank** [comment]
> Also can you please send similar PRs to website repository so I can comment on it too. Thank you!

Thank you very much for giving me this opportunity to participate in the open source community work. I will mention some higher quality pr in the future.

File: cmd/kubelet/app/server.go (hunk 2 above)
Comment (SergeyKanzhelev) [reply cmd/kubelet/app/server.go:155]: you are correct, my mistake

**Review by SergeyKanzhelev — APPROVED** [review]
/ok-to-test
/lgtm

**Review by mrunalp — APPROVED** [review]

**Pull request merged**